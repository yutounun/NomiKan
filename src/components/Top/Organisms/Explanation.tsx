import React from "react";
import Cheers from "static/cheers.svg";

function Explanation() {
  return (
    <>
      <span className="text-gray top-title">部長のお会計をお助け！</span>
      <img src={Cheers} alt="" />
      <span className="explanation">
        各メンバーの支払額を%で計算することができます
      </span>
    </>
  );
}

export default Explanation;
