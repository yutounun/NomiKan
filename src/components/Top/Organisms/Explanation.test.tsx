import { render } from "@testing-library/react";
import React from "react";
import Explanation from "./Explanation";

describe("Explanation", () => {
  it("takes a snapshot", () => {
    const { asFragment } = render(<Explanation />);
    expect(asFragment()).toMatchSnapshot();
  });
});
