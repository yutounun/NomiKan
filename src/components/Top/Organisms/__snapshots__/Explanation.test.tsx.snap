// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Explanation takes a snapshot 1`] = `
<DocumentFragment>
  <span
    class="text-gray top-title"
  >
    部長のお会計をお助け！
  </span>
  <img
    alt=""
    src="cheers.svg"
  />
  <span
    class="explanation"
  >
    各メンバーの支払額を%で計算することができます
  </span>
</DocumentFragment>
`;
